.travis/deploy_minikube.sh

```diff
@@ -14,15 +14,35 @@ set -x
 # NOTE: This script was originally copied from the Cojaeger-operator build
 # https://github.com/jaegertracing/jaeger-operator/blob/master/.travis/setupMinikube.sh
 
-# socat is needed for port forwarding
-sudo apt-get update && sudo apt-get install socat && sudo apt-get install conntrack
-
 export MINIKUBE_VERSION=v1.8.2
 export KUBERNETES_VERSION=v1.17.3
 
-sudo mount --make-rshared /
-sudo mount --make-rshared /proc
-sudo mount --make-rshared /sys
+source /etc/os-release
+
+if [ "${ID}" == "ubuntu" ]
+then 
+    # socat is needed for port forwarding
+    sudo apt-get update && sudo apt-get install socat && sudo apt-get install conntrack
+    sudo mount --make-rshared /
+    sudo mount --make-rshared /proc
+    sudo mount --make-rshared /sys
+elif [ "${ID}" == "centos" ]
+then
+    dnf install -y sudo
+    sudo dnf -y update && sudo dnf -y install socat conntrack 
+    dnf install -y dnf-plugins-core
+    sudo dnf config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+    sudo dnf install -y docker-ce docker-ce-cli containerd.io --nobest
+    #disable SELinux
+    SELinux_status=$(sestatus | grep "SELinux status" | awk -F ":" '{print $2}' | xargs )
+    if [ "${SELinux_status}" == "enabled" ]
+    then
+        sudo setenforce 0
```

Comment:
oy :(

Why?

Reply:
Currently, kubeadm on CentOS needs SELinux to be disabled:

https://github.com/kubernetes/minikube/issues/6014#issuecomment-563367526

Reply:
Does that happen with podman as well?

Reply:
(I mean, somehow, magically it is working in RHEL, no?)

Reply:
We should look into it one we can switch into podman (see previous comment https://github.com/noobaa/noobaa-core/pull/6073#discussion_r449873237)

Reply:
There are some locations where minikube stores executables, and these try to access configuration files. SElinux prevents that. I am not sure if the location that minikube uses is dynamic, or predictable. In the 2nd case, creating the directories in advance and setting appropriate labels might work.

This obviously need some more research. Moving to Permissive mode is acceptable for the moment.